simulation-runner/src/main/scala/com/puppetlabs/gatling/node_simulations/FOSS5xEmptyRepo.scala

```diff
@@ -0,0 +1,71 @@
urrent_mode%22%3A%22enforcing%22%2C%22selinux_enforced%22%3Atrue%2C%22selinux_policyversion%22%3A%2228%22%2C%22serialnumber%22%3A%22VMware-42%2031%201f%20be%208c%205d%203f%201c-57%20b9%2025%202d%20a7%2022%205c%20f7%22%2C%22ssh%22%3A%7B%22ecdsa%22%3A%7B%22fingerprints%22%3A%7B%22sha1%22%3A%22SSHFP%203%201%20678029dbd06737d013e8cded0e1222e0950c6b01%22%2C%22sha256%22%3A%22SSHFP%203%202%203d2bb6e0047c1c7ef99dee74696e15a1c813d53b8a3d92079dd12d9d9be669bb%22%7D%2C%22key%22%3A%22AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBHkvfwClxVvdxDc4ADuzKSkYwQx6udBqctS9XAXB6yEdduWApC%2BFWskpjLQs61mKeWO2wpIscDIrC2LRhnXNqd8%3D%22%7D%2C%22rsa%22%3A%7B%22fingerprints%22%3A%7B%22sha1%22%3A%22SSHFP%201%201%20abc203895fdafaf7becd0af7480f26587b0ac44a%22%2C%22sha256%22%3A%22SSHFP%201%202%2066b3da48acc21a37c8229d9327d8bc2c52e64210445710114db5352689c1f324%22%7D%2C%22key%22%3A%22AAAAB3NzaC1yc2EAAAADAQABAAABAQDFeLm%2B%2BlQ2XDRHcGuqOf%2B%2FhGGNiZy0MITk6AmqAszur3f344ZmrEH5tViqaNtw8JNe330ocUP2I1SKYnqdHExkHUcrPjfLpNrdForcMnAmc3Q%2Bn3DA98uyOWrnOu9FYegUuv7XaPK8uDokJAK3qiBu59XwAtysldpJ7kdQ2BAQpqIap%2FtgRpuRrNfpyscui5UfaD3dHvWuMHHxSk3a7oWBt7acm0IqKePxXeaSNX5IO2UUokAcm45jHHJzh7TnY9Q5PAe1XtdRESrB8ADGgDSgUhG4cjOnvBwMvfRLR29%2FAML3RDwk7r6z6ZPyaptifwm74gml%2FVXbeCWT0jAigFGn%22%7D%7D%2C%22sshecdsakey%22%3A%22AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBHkvfwClxVvdxDc4ADuzKSkYwQx6udBqctS9XAXB6yEdduWApC%2BFWskpjLQs61mKeWO2wpIscDIrC2LRhnXNqd8%3D%22%2C%22sshfp_ecdsa%22%3A%22SSHFP%203%201%20678029dbd06737d013e8cded0e1222e0950c6b01%5CnSSHFP%203%202%203d2bb6e0047c1c7ef99dee74696e15a1c813d53b8a3d92079dd12d9d9be669bb%22%2C%22sshfp_rsa%22%3A%22SSHFP%201%201%20abc203895fdafaf7becd0af7480f26587b0ac44a%5CnSSHFP%201%202%2066b3da48acc21a37c8229d9327d8bc2c52e64210445710114db5352689c1f324%22%2C%22sshrsakey%22%3A%22AAAAB3NzaC1yc2EAAAADAQABAAABAQDFeLm%2B%2BlQ2XDRHcGuqOf%2B%2FhGGNiZy0MITk6AmqAszur3f344ZmrEH5tViqaNtw8JNe330ocUP2I1SKYnqdHExkHUcrPjfLpNrdForcMnAmc3Q%2Bn3DA98uyOWrnOu9FYegUuv7XaPK8uDokJAK3qiBu59XwAtysldpJ7kdQ2BAQpqIap%2FtgRpuRrNfpyscui5UfaD3dHvWuMHHxSk3a7oWBt7acm0IqKePxXeaSNX5IO2UUokAcm45jHHJzh7TnY9Q5PAe1XtdRESrB8ADGgDSgUhG4cjOnvBwMvfRLR29%2FAML3RDwk7r6z6ZPyaptifwm74gml%2FVXbeCWT0jAigFGn%22%2C%22stage%22%3A%22prod%22%2C%22swapfree%22%3A%22820.00%20MiB%22%2C%22swapfree_mb%22%3A819.99609375%2C%22swapsize%22%3A%22820.00%20MiB%22%2C%22swapsize_mb%22%3A819.99609375%2C%22system_uptime%22%3A%7B%22days%22%3A0%2C%22hours%22%3A5%2C%22seconds%22%3A20007%2C%22uptime%22%3A%225%3A33%20hours%22%7D%2C%22timezone%22%3A%22PST%22%2C%22uptime%22%3A%225%3A33%20hours%22%2C%22uptime_days%22%3A0%2C%22uptime_hours%22%3A5%2C%22uptime_seconds%22%3A20007%2C%22uuid%22%3A%2242311FBE-8C5D-3F1C-57B9-252DA7225CF7%22%2C%22virtual%22%3A%22vmware%22%2C%22whereami%22%3A%22portland%22%2C%22clientcert%22%3A%22${node}%22%2C%22clientversion%22%3A%225.3.3%22%2C%22clientnoop%22%3Afalse%7D%2C%22timestamp%22%3A%222017-12-08T11%3A30%3A42.109494978-08%3A00%22%2C%22expiration%22%3A%222125-12-08T12%3A00%3A42.109796191-08%3A00%22%7D")
```

Comment:
Can we put the facts in an external file like we're doing with reportBody?

Reply:
I'm not sure... all of the other scala files all have the facts inline. I think the reportBody is in a special format that makes it different (based on what I could find about ELFileBody).

Reply:
That being said, I'm sure I could just read it in using whatever the regular file read function is in Scala. I'll take a look.

Reply:
If I can get that to work, I guess it might be worth looking into updating the script to do that as part of post-processing.